.github/workflows/ci.yaml

```diff
@@ -103,6 +103,19 @@ jobs:
         if: matrix.bzlmodEnabled
         run: echo "bzlmod_flag=--enable_bzlmod" >> $GITHUB_OUTPUT
 
+      - name: Set credential helpers flag
+        # Add --config needs_credential_helpers to add additional credential helpers
+        # to fetch from registries with HTTP headers set by credential helpers.
+        id: set_credential_helper_flag
+        if: matrix.bazelversion == '6.4.0' && matrix.folder == '.'
+        run: echo "credential_helper_flag=--config=needs_credential_helpers" >> $GITHUB_OUTPUT
+
+      - name: Setup credential helpers
```

Comment:
is this name correct? how is setup-crane doing credential helpers?

Reply:
Right, it could name it better, it just downloading crane to CI machine so that the credential helper can invoke it.

Reply:
okay fixed it,